src/helpers/validation.ts

```diff
@@ -0,0 +1,25 @@
+import validationRegExp from '../constants/validation'
+
+export const validateRegField = (field: any, value: any) => {
+    const {loginValidation, passwordValidation} = validationRegExp
+    switch(field){
+        case 'login':
+            if (!loginValidation.test(String(value).toLowerCase())) {
+                return 'Invalid login';
+            } else {
+                return '';
+            }
+        case 'password':
+            if (!passwordValidation.test(String(value).toLowerCase())) {
+                if (!value) {
```

Comment:
if (!passwordValidation.test(String(value).toLowerCase())) {
return !value ? `Password can't be empty` : 'Invalid Password'
}

return ''